plugins/meta/firewall/iptables.go

```diff
@@ -43,7 +43,16 @@ func ensureChain(ipt *iptables.IPTables, table, chain string) error {
 		}
 	}
 
-	return ipt.NewChain(table, chain)
+	err = ipt.NewChain(table, chain)
```

Comment:
LGTM for your intention and implementation, but IMO, idempotency should be done in `iptables` package because decoupling is important for us...  cc @squeed  waiting for your suggestion...

Reply:
It could be done there, sure. But the `iptables` package is a fairly bare-bones wrapper around the command line tools. They also don't have any documentation promising idempotency, whereas `containernetworking/plugins` does.

Reply:
Well, I know the maintainer :-).